lib/sdl/Makefile

```diff
@@ -40,3 +40,36 @@ clean-sdl_ttf:
 	           $(FREETYPE_OBJS) \
 	           $(NXDK_DIR)/lib/libSDL_ttf.lib \
 	           $(NXDK_DIR)/lib/libfreetype.lib
+
+LIBSDLIMG_DIR = $(NXDK_DIR)/lib/sdl/SDL2_image
```

Comment:
nit: I'm not a big fan of calling this SDLIMG / the sample sdl_img.
Is this a common alias for SDL_image, other than for their API prefix? This is the first time I see someone calling it like that, and I'm worried it will lead to potential confusion or errors later.

Reply:
I've renamed the sample and the variables.